src/converse-muc-views.js

```diff
@@ -467,6 +467,17 @@ converse.plugins.add('converse-muc-views', {
                 'input .chat-textarea': 'inputChanged',
                 'dragover .chat-textarea': 'onDragOver',
                 'drop .chat-textarea': 'onDrop',
+                'click .toggle-download': 'downloadAttachements',
+            },
+
+            downloadAttachements () {
+                var downloadable = this.el.getElementsByClassName('downloadable');
+                if (downloadable.length === 0) {
+                    this.showHelpMessages([__('There are no files to download.')], 'error');
+                    return;
+                }
+                _converse.multimediaDownloadModel = new _converse.MultimediaDownloadModel(downloadable, this.model.get('name'));
+                _converse.multimediaDownloadModel.downloadAttachements();
```

Comment:
This should go into src/converse-attachement-download.js and be added to the prototype via `Object.assign.`